tests/integration/src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala

```diff
@@ -140,8 +139,8 @@ class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonT
             endpoints = Seq(raml.Endpoint(name = "task1", hostPort = Some(0))),
             healthCheck = Some(MesosHttpHealthCheck(portIndex = Some(PortReference("task1")), path = Some("/ping"))),
             volumeMounts = Seq(
-              VolumeMount(Some("python"), s"$containerDir/python", false),
-              VolumeMount(Some("data"), s"$containerDir/data", true)
+              VolumeMount(Some("python"), s"$containerDir/python", true),
+              VolumeMount(Some("data"), s"$containerDir/data", false)
```

Comment:
@meichstedt and @timcharper, `true` marks the volume as read-only. Why did the tests pass on Mesos 1.7 and earlier? They should have always failed.